### DISCUSSION TITLE
Grammar issue - "Mark as play/watch"

### DISCUSSION THREAD

**Discussion by kaibagley** [IgnisDa/ryot#1069] [Q&A]
Hey, thought I'd open a discussion for this: https://github.com/IgnisDa/ryot/issues/820, since I think that small tweaks like this improve the overall feel of the app. 

Unfortunately I don't have any solutions I can think of right now outside of non-practical NLP stuff.. I'm not much of a programmer sorry. Is it feasible to simply hard-code in "mark as watched", "mark as read" and "mark as played" for movies/shows, books and games respectively?

And I know that Watchlist is a collection, but I also think this should be lower-case too instead of "Add to Watchlist"

**Comment by IgnisDa** [comment]
Unfortunately the solution you recommend is not very elegant. It would make the frontend more unreadable as well as make it more unmaintainable. As for the watchlist thing, I will change it to be lowercase.
